### ISSUE TITLE
a11y: Improve toast accessibility

### ISSUE THREAD

**Issue by desig9stein** [IgniteUI/igniteui-angular#8931]
## Description  
Provide an accessible experience for igx-toast.

`aria-live` should be added to the message and by default should be `polite`.
The user should be able to change that, for example, `aria-live="assertive"` or `aria-live="off"`

 * igniteui-angular version: 11.1.x
 * browser: all

## Attachments  
Attach a sample if available, and screenshots, if applicable.
